src/exception.rs

```diff
@@ -103,6 +103,11 @@ pub fn handle_exception_sync(ctx: &mut TrapFrame) -> AxResult<AxVCpuExitReason>
             // The `#imm`` argument when triggering a hvc call, currently not used.
             let _hvc_arg_imm16 = ESR_EL2.read(ESR_EL2::ISS);
 
+            // Is this a psci call?
+            if let Some(result) = handle_psci_hvc(ctx) {
```

Comment:
Is there any other method for distinguishing HVC and SMC here?

Reply:
```C
    else if (ec == 0x16)
    { // hvc
        print_info("            This is hvc call handler\n");
        return;
    }
    else if (ec == 0x17)
    { // smc
        ept_violation_info_t info;
        //printf("Prefetch abort : %x\n", hsr.bits);
        info.hsr.bits = hsr.bits;
        print_info("            This is smc call handler\n");
        advance_pc(&info, ctx_el2);
        return;
    }
```
ESR_EL2::EC::  <=> ec

Reply:
@aarkegz

Reply:
Yes, but the questions here is that SMC is not trapped now. (`HCR.TSC` is not set, it was but later this line was commented out `src/vcpu.rs:191`, we need to figure out why in the first place.)

Reply:
Handling HVCs only is a good workaround for the current situation. It's minimal and can be easily extended to a full implementation.

Reply:
> Yes, but the questions here is that SMC is not trapped now. (`HCR.TSC` is not set, it was but later this line was commented out `src/vcpu.rs:191`, we need to figure out why in the first place.)

Yep, I think that we also need to modify arceos's [boot.rs](https://github.com/arceos-hypervisor/arceos/blob/8f55599a4c9caf1c654564668e51b20fc939bc67/modules/axhal/src/platform/aarch64_common/boot.rs#L208)